test/relations.test.js

```diff
@@ -2453,9 +2621,17 @@ describe('relations', function() {
       if (!author) return done();
       Author.findById(author.id, function(err, author) {
         author.pictures(function(err, pics) {
-          pics.should.have.length(2);
```

Comment:
because the delete in the previous test was not executed due to missing feature.